src/components/30-organisms/footer/index.scss

```diff
@@ -0,0 +1,314 @@
+      }
+    }
+
+    &-title {
+      @include typo(primary, medium, bold);
+
+      display: none;
+
+      text-transform: uppercase;
+      line-height: 1.5;
+
+      @include breakpoint($mediaquery-lg-up) {
+        display: block;
+      }
+    }
+
+    &-list {
+      margin: 0;
+      text-indent: 0;
+      list-style-type: none;
+      padding: 0;
+
+      display: flex;
+      flex-wrap: wrap;
+      padding: 15px 20px;
+
+      @include breakpoint($mediaquery-sm-up) {
+        margin: 0;
+        text-indent: 0;
+        list-style-type: none;
+        padding: 0;
+
+        display: flex;
+        flex-wrap: wrap;
+        padding-top: 15px;
+      }
+
+      @include breakpoint($mediaquery-lg-up) {
+        padding: 10px 0;
+        margin-left: -10px;
+      }
+
+      @include breakpoint($mediaquery-sm-up) {
+        a {
+          display: block;
+          padding: 10px 10px 0 10px;
+          margin-right: 10px;
+        }
+
+        a:first-of-type {
+          padding: 10px 10px 0 0;
+        }
+      }
+    }
+  }
+
+  &__title-desktop {
+    display: none;
+
+    @include breakpoint($mediaquery-sm-up) {
+      display: block;
+      @include typo(primary, medium, bold);
+
+      text-transform: uppercase;
+      line-height: 1.5;
+    }
+  }
+
+  &__accordion-button {
+    @include typo(primary, default, medium);
+    display: flex;
+    justify-content: space-between;
+    color: $color-prim-white;
+    background-color: $color-prim-blue-dark-indigo;
+    padding: 20px 30px 20px 30px;
+    width: 100%;
+    text-align: left;
+
+    border: none;
+    outline: 0;
+
+    border-bottom: 1px solid $color-prim-blue-st-tropaz;
+    text-transform: uppercase;
+
+    svg {
+      width: 10px;
+      height: 10px;
+      color: $color-prim-gray-dusty;
+    }
+
+    &-caret {
+      transition: transform 0.2s ease-in-out;
+      transform: rotate(180deg);
+
+      &--open {
+        transform: rotate(0deg);
+      }
+    }
+
+    @include breakpoint($mediaquery-sm-up) {
+      display: none;
+    }
+  }
+
+  &__main {
+    @include breakpoint($mediaquery-sm-up) {
+      margin: 0 10px;
+    }
+
+    &-content-panel {
+      margin: 0;
+      text-indent: 0;
+      list-style-type: none;
+      padding: 0;
+
+      max-height: 0;
+      transition: max-height 0.2s ease-out;
+
+      overflow: hidden;
+      background-color: $color-prim-blue-azure;
+
+      &--open {
+        max-height: 456px;
+      }
+
+      &-list-item {
+        @include typo(primary, xx-small, default);
+
+        padding: 0;
+        border-bottom: 1px solid $color-prim-blue-dark-indigo;
+
+        &:last-child {
+          border-bottom: none;
+        }
+
+        ::slotted(a) {
+          display: block;
+          text-decoration: none;
+          color: $color-prim-white;
+          padding: 20px 30px !important; // Needed to dominate body reset
+        }
+
+        ::slotted(a:hover) {
+          color: rgba(255, 255, 255, 0.65);
```

Comment:
undeclared color, shouldn't this be added to global colors after discussion with designers?

Reply:
Yes, I agree.

Reply:
I created an issue for this request: https://github.com/axa-ch/patterns-library/issues/1109